plugins/inputs/nsdp/README.md

```diff
@@ -0,0 +1,71 @@
+# NSDP (Netgear Switch Discovery Protocol) Input Plugin
+
+This input plugin gathers status from NSDP
+([Netgear Switch Discovery Protocol][1]) capable devices.
+
+[1]: https://en.wikipedia.org/wiki/Netgear_Switch_Discovery_Protocol
+
+Retrieved status are:
+
+- Bytes (sent and received)
+- Total amount of packates processed
+- Total amount of broadcasts processed
+- Total amount of multicast processed
+- Total amount of errors occurred
+
+Each per switch and port.
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
+
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
+
+## Configuration
+
+```toml @sample.conf
+# Gather NSDP (Netgear Switch Discovery Protocol) status
+[[inputs.nsdp]]
+  ## The target address to use for NSDP gathering the status.
+  # target = "255.255.255.255:63322"
```

Comment:
How about naming this `address`? Or is `target` a well-known identifier in this domain?

Reply:
Adapted via 8c321f127f426a5c1cbdc9876b56cf4409fafce2